docs/intro.md

```diff
@@ -75,6 +75,7 @@ You can build this book locally on the command line via the following steps:
 
     ```
     cd quantecon-mini-example
+    jupyter-book toc mini_book/docs
```

Comment:
The problem here isn't that a `toc` needs to be created, but that the command here points to the wrong folder, so instead the `build` command below should be `jupyter-book build mini_book`